build/goreleaser/linux/sles_156_arm.yml

```diff
@@ -0,0 +1,77 @@
+  # SLES 15.6 arm
+  
+  - id: sle-15.6-infrastructure-agent-arm
+    builds:
+      - linux-agent-arm
+      - linux-ctl-arm
+      - linux-service-arm
+    package_name: newrelic-infra
+    file_name_template: "newrelic-infra-{{ .Env.TAG }}-1.sles15.6.{{ .Arch }}"
+    vendor: 'New Relic, Inc.'
+    homepage: 'https://docs.newrelic.com/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes'
+    maintainer: 'caos-team@newrelic.com'
+    description: 'New Relic Infrastructure provides flexible, dynamic server monitoring. With real-time data collection and a UI that scales from a handful of hosts to thousands, Infrastructure is designed for modern Operations teams with fast-changing systems.'
+    license: 'Copyright (c) 2008-2021 New Relic, Inc. All rights reserved.'
+    formats:
+      - rpm
+    bindir: /usr/bin
+    contents:
+#      - src: 'assets/examples/logging/linux/file.yml.example'
```

Comment:
should this be removed or adding a comment why is it commented?

Reply:
I guess I'll just remove them xD